lib/aws-record/record/attribute.rb

```diff
@@ -48,7 +48,8 @@ class Attribute
       #   item is nil or not set at persistence time.
       def initialize(name, options = {})
         @name = name
-        @database_name = options[:database_attribute_name] || name.to_s
+        @database_name = (options[:database_attribute_name] &&
```

Comment:
Does `(options[:database_attribute_name] || name).to_s` simplify things here?

Reply:
Yeah - that works.  Much cleaner, good suggestion.